common/components/common/groups/AboutGroupDisplay.jsx

```diff
@@ -68,7 +69,7 @@ class AboutGroupDisplay extends React.Component<Props, State> {
     return this.state.group ? (
       this._renderPageLayout()
     ) : (
-      <div>{this.state.loadStatusMsg}</div>
+      <LoadingFrame height={600} />
```

Comment:
I think this conditional check may be irrelevant because the AboutGroupController also has conditional checks - that is, `this.state.group` will always be true because this entire component isn't rendered until the group data has returned from the API. The same is true of AboutProjectController / AboutProjectDisplay. See the changes in commit 1d01dd5. 

If that's the case I can just remove this conditional here and in AboutProjectDisplay, clean those up a bit.

It may be worth the time (possibly in another PR) to refactor out AboutProjectController and AboutGroupController to remove helmet, produce error messages in another fashion, and

Reply:
Yeah, this check is done by the parent in all the current cases.